.github/workflows/build-bundle-linux.yml

```diff
@@ -0,0 +1,60 @@
+name: Build and Run tests
+
+on: [push]
```

Comment:
Where will the build put the artifacts?

Reply:
Artifacts handling is under investigation, as  honestly said, just start learning :)

Reply:
According to https://github.com/actions/upload-artifact

Something like this 
```
- uses: actions/upload-artifact@v1
  with:
    name: linux packages
    path: ${{runner.workspace}}/build/packaging
```

I need to update build scripts further to produce artifacts under build/packaging/ folder. (currently under build folder where are with other output files)

Reply:
Implemented.